webapps-2019.html

```diff
@@ -187,11 +187,10 @@ <h2>
           </li>
         </ul>
         <p>
-          For <a href="#background">historical reasons</a>, the working group
-          maintains a specification for mapping HTML elements and attributes to
+          The working group also maintains a specification for mapping HTML elements and attributes to
```

Comment:
"The Working Group" is duplicated on this line (from the preceding line) I think?

Reply:
The whole paragraph reads: "The working group also maintains a specification for mapping HTML elements and attributes to platform accessibility APIs, and a separate specification that defines author conformance requirements for setting ARIA attributes. The Working Group does not expect to add any other specifications relating to this matter.". I could remove the second occurrence and replace with "The Group" if you'd like. But otherwise, it sounds fine to me.